sandbox.yml

```diff
@@ -0,0 +1,76 @@
+version: "3.6"
+services:
+  db:
+    container_name: kepler-tzkt-db
+    restart: always
+    image: postgres:13
+    environment:
+      POSTGRES_USER: tzkt
+      POSTGRES_PASSWORD: qwerty
+      POSTGRES_DB: tzkt_db
+    volumes:
+      - postgres:/var/lib/postgresql/data
+
+  api:
+    container_name: kepler-tzkt-api
+    restart: always
+    image: bakingbad/tzkt-api:latest
+    depends_on:
+      - db
+      - sync
+    ports:
+      - 5000:5000
+
+  sync:
+    container_name: kepler-tzkt-sync
+    restart: always
+    image: bakingbad/tzkt-sync:latest
+    environment:
+      - TZKT_TEZOSNODE__ENDPOINT=http://sandbox:20000/
+      - TZKT_TEZOSNODE__CHAINID=NetXfpUfwJdBox9
+    depends_on:
+      - db
+      - sandbox
+
+  sandbox:
+    container_name: kepler-tzkt-sandbox
+    restart: always
+    image: tqtezos/flextesa:20210602
+    command: flobox start
+    environment:
+      - flextesa_node_cors_origin=*
+    ports:
+      - 8732:20000
+
+  kepler1:
+    container_name: kepler-1
+    restart: always
+    image: kepler:latest
```

Comment:
is there a way to have a single image be built and used for two containers? `build: .` works but will build the image twice

Reply:
Looks like only the first one needs to having the `build` instruction https://stackoverflow.com/a/41597317 And then you can use a very specific tag like `dockercompose-testing` to avoid pulling a published version unintentionally.